apps/activity/terraform/aws/main.tf

```diff
@@ -1,23 +1,36 @@
-variable "user_name" {}
+variable "role_name" {}
+variable "profile_name" {}
 
-resource "aws_iam_user" "aws_user" {
-  name = "${var.user_name}-user"
-  path = "/system/"
+resource "aws_iam_role" "cloud_activity_role" {
+  name = "${var.role_name}"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
```

Comment:
With no other entitlements does this just generate only errors? Not sure if we want to give it rights to enumerate _some_ resources

Reply:
I created an instance in my own personal AWS account using the existing plan.  I then created a second instance with this new plan.  I tested the CLI commands in both to make sure they matched.  Basically all the AWS CLI commands failed except for these:
   aws ec2 describe-instances
   aws sts get-caller-identity
   aws iam get-user
We don't give the user any special permission either.